packages/icons/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/icons",
-  "version": "1.112.1",
+  "version": "1.112.2",
```

Since this is adding a new icon and not fixing a bug, it should be a minor release, not a patch
```suggestion
  "version": "1.113.0",
```